kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/JbpmKieServerExtension.java

```diff
@@ -474,6 +483,51 @@ protected void addAsyncHandler(KModuleDeploymentUnit unit, InternalKieContainer
         }
     }
 
+    protected void addTaskBAMEventListener(final KModuleDeploymentUnit unit, final InternalKieContainer kieContainer) {
+        final DeploymentDescriptor descriptor = getDeploymentDescriptor(unit, kieContainer);
+        descriptor.getBuilder().addTaskEventListener(
+                new NamedObjectModel(
```

Comment:
It's enough to use ObjectModel instead of NamedObjectModel as named is used for objects that needs to be registred with given name like work item handlers ir globals.